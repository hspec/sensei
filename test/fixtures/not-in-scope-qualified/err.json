{"version":"1.0","ghcVersion":"ghc-9.10.0","span":{"file":"test/fixtures/not-in-scope-qualified/Foo.hs","start":{"line":2,"column":7},"end":{"line":2,"column":12}},"severity":"Error","code":76037,"message":["Not in scope: `B.c2w'\nNB: no module named `B' is imported."],"hints":[]}
