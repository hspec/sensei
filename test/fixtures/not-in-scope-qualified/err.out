test/fixtures/not-in-scope-qualified/Foo.hs:2:7: error: [GHC-76037]
    Not in scope: `M.catMaybes'
    NB: no module named `M' is imported.

